### ISSUE TITLE
Misleading documentation for PMPCorefinementVisitor?

### ISSUE THREAD

**Issue by Simon-Lopez** [CGAL/cgal#6842]
In the documentation of [PMPCorefinementVisitor::intersection_point_detected](https://doc.cgal.org/latest/Polygon_mesh_processing/classPMPCorefinementVisitor.html#a00ee0ca85db535a48726a92414acda7f) it is said:

* is_target_coplanar is true iff the target of h_e is the intersection point*

I have not checked the code of PMP::corefine but using it with a visitor and CGAL 5.4.2 it seems that *h_e* should be replaced by *h_f*.

@sloriot could you confirm or infirm this, please ? Thanks a lot !

**Comment by sloriot** [comment]
No, that's correct. What makes you think it's not the case? If the intersection point is a point of `tm_f`, sdim==0 and it is  the target of `h_f`.
